internal/learn/future/shimmer.md

```diff
@@ -1,37 +1,30 @@
 ---
 id: shimmer
 title: Shimmer
-description: Introduction to the emerging incentivized staging network
+description: Introduction to Shimmer - the incentivized staging network
 keywords:
-  - Incentivized
-  - staking
-  - SMR
+- Incentivized
+- staking
+- SMR 
 ---
+# Future of Shimmer
 
 ![Shimmer](/img/learn/future/shimmer.svg 'Click to see the full-size image.')
 
-In November 2021, IOTA Foundation [introduced](https://blog.iota.org/introducing-iota-staking/) Shimmer, an incentivized staging network. Shimmer will function as a validation and staging network for upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token). To fairly distribute Shimmer tokens on the new network, you can stake your IOTA tokens before the launch of Shimmer, and will receive Shimmer tokens once the new network starts.
+In November 2021, the IOTA Foundation [introduced Shimmer](https://blog.iota.org/introducing-iota-staking/), an incentivized staging network. Shimmer will function as a validation and staging network for upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token). To fairly distribute Shimmer tokens on the new network, IOTA holders [could stake their tokens](https://blog.iota.org/iota-staking-start/) to receive Shimmer tokens once the new network starts. You may verify the results of the 90-day pre-launch staking the [staking event overview JSON file](https://github.com/iotaledger/participation-events/blob/master/events/staking/shimmer.json) and [staking results JSON file](https://raw.githubusercontent.com/iotaledger/participation-events/master/results/staking/shimmer.json).
```

Comment:
The last sentence seems to be missing some wording. I guess `using`?

Reply:
```suggestion
In November 2021, the IOTA Foundation [introduced Shimmer](https://blog.iota.org/introducing-iota-staking/), an incentivized staging network. Shimmer will function as a validation and staging network for upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token). To fairly distribute Shimmer tokens on the new network, IOTA holders [could stake their tokens](https://blog.iota.org/iota-staking-start/) to receive Shimmer tokens once the new network starts. You may verify the results of the 90-day pre-launch staking using the [staking event overview JSON file](https://github.com/iotaledger/participation-events/blob/master/events/staking/shimmer.json) and [staking results JSON file](https://raw.githubusercontent.com/iotaledger/participation-events/master/results/staking/shimmer.json).
```